tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -48,13 +52,14 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(value = ClusterTestExtensions.class)
-@ClusterTestDefaults(clusterType = Type.ZK)
+@ClusterTestDefaults(clusterType = Type.ALL)
 @Tag("integration")
 public class GetOffsetShellTest {
     private final int topicCount = 4;
     private final int offsetTopicPartitionCount = 4;
     private final ClusterInstance cluster;
     private final String topicName = "topic";
+    private final Duration consumerTimeout = Duration.ofMillis(100);
```

Comment:
Could you increase the timeout to run QA again? The failed tests are caused that `__consumer_offsets` not created.

Reply:
Sure. Bumped the timeout to 1000.